static/js/jira-configuration.js

```diff
@@ -34,39 +34,47 @@ $('.configure-connection-link').click(function (event) {
 $('.delete-connection-link').click(function (event) {
   event.preventDefault()
 
-  $.ajax({
-    type: 'DELETE',
-    url: `/jira/configuration?jwt=${encodeURIComponent(params.get('jwt'))}&xdm_e=${encodeURIComponent(params.get('xdm_e'))}`,
-    data: {
-      installationId: $(event.target).data('installation-id')
-    },
-    success: function (data) {
-      AP.navigator.reload()
-    }
-  })
+	window.AP.context.getToken(function(token){
```

Comment:
So for our frontend calls we'll now use Context tokens instead of reusing the one passed with iframe query